src/components/Filter/Filter.js

```diff
@@ -39,6 +41,13 @@ export default class Filter extends Component {
     this.props.onToggleFilter();
   };
 
+  onToggleSwitch = () => {
+    this.setState({
```

Comment:
Correct me if I'm wrong, but due `state.showFavs` is not been used in the "view" we don't need it in the `state`. If I understand, it meant only for passing it to the callback `props.onToggleSwitch`. If so, [`onToggleSwitch`](https://github.com/Coding-Coach/find-a-mentor/pull/201/files#diff-c9fd7e5f49ae423156ad6951387e9f61R44) gets [an `event` from the checkbox](https://github.com/Coding-Coach/find-a-mentor/pull/201/files#diff-d7fe704bea926ddd248d59674f7cdd1eR13) so it can be check whether the input is checked or not.

WDYT?

Reply:
oh my bad, yes you are right, we do not need it in the `state`